ci/code_checks.sh

```diff
@@ -12,9 +12,10 @@
 #   $ ./ci/code_checks.sh doctests      # run doctests
 #   $ ./ci/code_checks.sh docstrings    # validate docstring errors
 #   $ ./ci/code_checks.sh single-docs   # check single-page docs build warning-free
+#   $ ./ci/code_checks.sh notebooks     # check correctness of documentation of notebooks
```

Comment:
rather than 'correctness', which isn't really well-defined (😄 ), how about "check execution of documentation notebooks"?

Reply:
> rather than 'correctness', which isn't really well-defined (😄 ), how about "check execution of documentation notebooks"?

Sure. Changed to the better wording in the second commit. Thank you, @MarcoGorelli